tools/c7n_gcp/tests/data/flights/autoscaler-query/get-compute-v1-projects-cloud-custodian-aggregated-autoscalers_1.json

```diff
@@ -0,0 +1,1009 @@
+{
+  "body": {
+    "items": {
+      "zones/europe-west3-b": {
+        "warning": {
```

Comment:
generally for aggregated results, the results can be slimmed down vastly by removing the extraneous empty page responses for regions/zones not of interest.

Reply:
Done.